### PR TITLE
Update to Rust 1.16.0

### PR DESCRIPTION
**Pull request by tbu-**

### CODE REVIEW

**Comment by tbu-** [comment]
@homunkulus try

**Comment by homunkulus** [comment]
:hourglass: Trying commit ee2b2d3 with merge ee2b2d3...

File: src/lib.rs [hunk 1]
```diff
@@ -143,42 +146,42 @@ pub use core_collections::vec;
...
 #[macro_use]
 mod macros;
 
 pub mod sync;
 
-// Rust 1.15.0
+// Rust 1.16.0
 pub mod ascii;
-// Rust 1.15.0
+// Rust 1.16.0 (incomplete)
```
Comment (japaric) [inline src/lib.rs:157]: Good idea 👍

**Review by japaric — APPROVED** [review]
Thanks, @tbu-. Feel free to land whenever. (I don't know if you want to land this before #110 or after that.)

@homunkulus delegate=tbu-

**Comment by homunkulus** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/japaric/steed/builds/222247307)
State: approved= try=True

**Comment by tbu-** [comment]
@homunkulus r+

**Comment by homunkulus** [comment]
:key: Insufficient privileges

**Comment by tbu-** [comment]
@homunkulus r=japaric

**Comment by homunkulus** [comment]
:key: Insufficient privileges

**Comment by japaric** [comment]
@homunkulus r+

**Comment by homunkulus** [comment]
:pushpin: Commit 3663209 has been approved by `japaric`

**Comment by japaric** [comment]
(Merging manually since the tests passed)

**Pull request merged**